analysis/analysis_test.go

```diff
@@ -50,8 +50,16 @@ func TestApplicationAnalysis(t *testing.T) {
 		t.Fatalf("Failed to create temp dir for ci repo: %v", err)
 	}
 	// Clone the konveyor/ci repository into the temp directory
-	cloneCmd := fmt.Sprintf("git clone https://github.com/konveyor/ci %s", ciTempDir)
-	cloneErr := utils.RunShellCommand(cloneCmd)
+	ciAuthor := os.Getenv("CI_REPO_AUTHOR")
+	if ciAuthor == "" {
+		ciAuthor = "konveyor"
+	}
+	ciBranch := os.Getenv("CI_REPO_BRANCH")
+	if ciBranch == "" {
+		ciBranch = "main"
+	}
+	ciRepoURL := fmt.Sprintf("https://github.com/%s/ci", ciAuthor)
```

Comment:
If we don't use submodules for linking CI repo, I would prefer allow customize full repo URL, instead just the author in github path. (e.g. if it would ever run somewhere internally).